ocaml/idl/datamodel_common.ml

```diff
@@ -10,7 +10,7 @@ open Datamodel_roles
               to leave a gap for potential hotfixes needing to increment the schema version.*)
 let schema_major_vsn = 5
 
-let schema_minor_vsn = 779
+let schema_minor_vsn = 781
```

Comment:
780 is used upstream or something like that?

Reply:
In fact we have a patch in our list of patches in the RPM that set it to 780. As in fact this patch is not based on the 24.19.2 but on 24.19.2 + our patches from RPM repo it explains the difference. I started with a patch on top of 24.19.2 but it has conflict that's why I rebase on 24.19.2 + our patches. Personally I'd like to create a branch in this repo based on 24.19.2 where we have our patches. So we will be able to generate patches for RPM using git format patches. Like it is done for SM I think.